core/src/main/resources/error/error-classes.json

```diff
@@ -256,6 +256,18 @@
       "Key <keyValue> does not exist. Use `try_element_at` to tolerate non-existent key and return NULL instead. If necessary set <config> to \"false\" to bypass this error."
     ]
   },
+  "MELT_REQUIRES_VALUE_COLUMNS" : {
```

Comment:
Given that even pandas using UNPIVOT in the description. Can we use that verb instead of met?https://pandas.pydata.org/docs/reference/api/pandas.melt.html

Reply:
+1

Reply:
You are saying use `UNIVOT` instead of `MELT` in this line only? Or use `unpivot` wherever this PR mentions `melt`? API, catalyst, docu?

In the latter case: should we at least have `Dataset.melt` as an alias for `Dataset.unpivot`, given in Pandas and PySpark Pandas the method *is called*  `melt`?

Reply:
I think the problem is which one should be the primary name? I'd vote for `unpivot` as it's a direct matching of the existing `pivot` feature in Spark.

I'm OK to add `Dataset.melt` as an alias of `Dataset.unpivot` for Pandas compatibility.